src/Parameters/CreateMeetingParameters.php

```diff
@@ -373,6 +373,11 @@ class CreateMeetingParameters extends MetaParameters
      */
     protected $userCameraCap;
 
+    /**
+     * @var array
```

Comment:
```suggestion
     * @var array<array{id: string, name: string, roster: string}>
```

Reply:
as roster is an array, I changed the type